site/source/docs/getting_started/FAQ.rst

```diff
@@ -60,9 +61,9 @@ Why is code compilation slow?
 =============================
 
 Emscripten makes some trade-offs that make the generated code faster and
-smaller, at the cost of longer compilation times. For example, we build parts of
-the standard library along with your code, which enables some additional
-optimizations, but takes a little longer to compile.
+smaller, at the cost of longer link times. For example, we build parts of
+the standard library with `-flto` (Link Time Optimization), which enables some
+additional optimizations, but can take a little longer to build.
```

Comment:
```suggestion
additional optimizations, but can take a little longer to build. We also
(in optimized builds) run the Binaryen optimizer on the entire output
even without LTO.
```

Reply:
This adds the more common reason for slower link times (as LTO is more rarely used).